core/src/main/java/de/neofonie/styla/core/models/CloudServiceModel.java

```diff
@@ -75,11 +75,15 @@ private String getCloudServiceProperty(String property) {
         return propertyValue;
     }
 
-    private String getCloudServiceProperty(ResourceResolver resourceResolver, Page contentRootPage) {
+    private String getCloudServiceProperty(final ResourceResolver resourceResolver, final Page page) {
         final String property = "seoApiUrl";
-        final String cloudServiceConfiguration = getCloudServiceConfiguration(contentRootPage);
+        final String cloudServiceConfiguration = getCloudServiceConfiguration(page);
 
-        if (!StringUtils.isNotEmpty(cloudServiceConfiguration)) {
+        if (StringUtils.isEmpty(cloudServiceConfiguration)) {
+            final Page parentPage = page.getParent();
+            if (parentPage != null) {
+                return getCloudServiceProperty(resourceResolver, parentPage);
+            }
             LOGGER.warn(String.format("Failed to get cloud service property: %s", property));

```

Comment:
this should be `error` instead of `warning`, as the page is broken if this doesn't work.